sdk/bcs/CHANGELOG.md

```diff
@@ -8,6 +8,12 @@
 
 ## 0.4.0
 
+- Adds support for generics;
```

Comment:
We shouldn't really manually modify this, it should be modified through changesets, so I think really we need to keep that in mind when we write changesets.

Reply:
I understand, and my main motivation for this one and for the future is people who depend on this package. As soon as more people get acquainted with the tech, we’ll have to inform about breaking changes and this one is huge.

Reply:
Do you want me to roll back this change? Does it break pnpm? Or we’re good?

Reply:
I think this is fine to keep, but we should try to avoid this in the future